In [2]:
import numpy as np
from scipy import stats
from scipy.stats import norm

Q. 5.	Consider a company that has two different divisions. The annual profits from the two divisions are independent and have distributions Profit1 ~ N(5, 32) and Profit2 ~ N(7, 42) respectively. Both the profits are in $ Million. Answer the following questions about the total profit of the company in Rupees. Assume that $1 = Rs. 45

In [3]:
mean = 5+7
print('Mean profit is Rs.', mean*45, 'Million')

Mean profit is Rs. 540 Million


In [4]:
SD = np.sqrt((9)+(16))
print('Standerd Deviation is Rs.', SD*45, 'Million')

Standerd Deviation is Rs. 225.0 Million


In [5]:
print('Range is Rs.', (stats.norm.interval(0.95,540,225)), 'in Millions')

Range is Rs. (99.00810347848784, 980.9918965215122) in Millions


In [6]:
X = 540+(-1.5)*(225)
print('5th peercentile of profit (in million Rupees) is', np.round(X))

5th peercentile of profit (in million Rupees) is 202.0


In [7]:
stats.norm.cdf(0,5,3)

0.0477903522728147

In [8]:
stats.norm.cdf(0,7,4)

0.040059156863817086

A) Range is Rs (99.00810347848784, 980.9918965215122) in Millions

B) To compute 5 th percentile ,we use the formula X = μ + Zσ; wherein
from z table, 5 th percntile = -1.5 X= 540+(-1.5) * (225)
Therefore, 5th peercentile of profit is 202.0 million Rupees

C)Probability of Division 1 making a loss P(X<0)
0.047
Probability of Division 2 making a loss P(X<0)
0.040